openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -25,6 +25,12 @@ class CourseOverview(django.db.models.Model):
     a course as part of a user dashboard or enrollment API.
     """
 
+    # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
+    VERSION = 1
```

Comment:
@jibsheet @adampalay The whole point of the versioning is that we wouldn't ever have to manually kill the table. Whenever there's a change that may screw up data, the entire cache is invalidated.